SpatialGDK/Source/Public/SpatialConstants.h

```diff
@@ -30,6 +37,9 @@ namespace SpatialConstants
 	const UnrealObjectRef NULL_OBJECT_REF(0, 0);
 	const UnrealObjectRef UNRESOLVED_OBJECT_REF(0, 1);
 
+	static const TCHAR* ClientWorkerType = TEXT("UnrealClient");
```

Comment:
Nit: can these be `const FString`?

Reply:
I didn't want to make these objects as they may cause global initialization order problems.

Reply:
Valentyn showed heavy existing usage of global FStrings in the Unreal codebase, so will switch.